packages/wrangler/src/config/index.ts

```diff
@@ -57,6 +57,17 @@ export function readConfig<CommandArgs>(
 		throw new UserError(diagnostics.renderErrors());
 	}
 
+	const mainModule = ("script" in args ? args.script : config.main) as string;
+	if (typeof mainModule === "string" && mainModule.endsWith(".py")) {
+		// Workers with a python entrypoint should have bundling turned off, since all of Wrangler's bundling is JS/TS specific
+		config.no_bundle = true;
+
+		// Workers with a python entrypoint need module rules for "*.py". Add one automatically as a DX nicety
+		if (!config.rules.find((rule) => rule.type === "PythonModule")) {
+			config.rules.push({ type: "PythonModule", globs: ["**/*.py"] });
+		}
```

Comment:
Do we need to add a rule for requirements.txt or is that only used locally?

Reply:
It can't really leverage the rules system in the same way as .py modules (since it's one file that has to be parsed and then turned into multiple modules)